libs/components/auto-scale/auto-scale.component.ts

```diff
@@ -15,9 +15,6 @@ export class AutoScaleComponent implements OnInit {
   @Input()
   height = 100;
 
-  constructor() { }
-
-  ngOnInit(): void {
-  }
-
+  @ContentChild(ElementRef)
+  private _elem: ElementRef;
```

Comment:
I think this can be removed completely since nothing uses it

Reply:
Yeah I just noticed aswell. First try to get the inner Element..